### PR TITLE
v2 2 0 release notes

### PR DESCRIPTION
**Pull request by ethanlcz**

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
Security Check : d1f82c15d6b6e69d448a1ef5094b5fc129d4e60c

**Comment by skylarCai** [comment]
auto build success,number:203

**Comment by skylarCai** [comment]
autotest fail,number:1541

**Review by wushengtws — APPROVED** [review]

**Comment by skylarCai** [comment]
Security Check : d1f82c15d6b6e69d448a1ef5094b5fc129d4e60c

File: haas_lib_bundles/python/README.md [hunk 1]
```diff
@@ -1,86 +1,5 @@
 # HaaS Python
```
Comment (skylarCai) [inline haas_lib_bundles/python/README.md:1]: LGFM

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**